pkg/grpc/config/config.proto

```diff
@@ -149,7 +149,7 @@ message Settings {
   optional string certificate_authority_file = 35;
   optional string signing_key = 36;
   optional string signing_key_algorithm = 62;
-  map<string, string> headers = 69;
+  map<string, string> set_response_headers = 69;
```

Comment:
Just double checking that this is fine to rename?

Reply:
yes it should be. protobuf is purely by the field id unless we use reflection